RealmSwift/Object.swift

```diff
@@ -260,6 +260,10 @@ public final class DynamicObject : Object {
     public override func setValue(value: AnyObject?, forUndefinedKey key: String) {
         self[key] = value
     }
+
+    @objc private class func shouldPersistToRealm() -> Bool {
+        return false;
```

Comment:
Why would we ever want to save DynamicObject?

-Samuel E. Giddins

On Apr 23, 2015, at 5:32 PM, Thomas Goyne <notifications@github.com> wrote:

In RealmSwift/Object.swift:

> @@ -260,6 +260,10 @@ public final class DynamicObject : Object {
>      public override func setValue(value: AnyObject?, forUndefinedKey key: String) {
>          self[key] = value
>      }
> +
> +    @objc private class func shouldPersistToRealm() -> Bool {
> +        return false;
This looks blatantly wrong. Is it somehow not?

—
Reply to this email directly or view it on GitHub.